includes/wf_crm_admin_form.inc

```diff
@@ -1712,15 +1719,17 @@ class wf_crm_admin_form {
                 );
                 // Cannot use isNewFieldset effectively.
                 $previous_data = $handler_configuration['settings'];
-                list(, $c, $ent) =  wf_crm_explode_key($key);
+                list(, $c, $ent) =  Utils::wf_crm_explode_key($key);
                 $type = in_array($ent, self::$fieldset_entities) ? $ent : 'contact';
                 $create = !isset($previous_data['data'][$type][$c]);
                 /*
     list(, $c, $ent) =  wf_crm_explode_key($field_key);
     $type = in_array($ent, self::$fieldset_entities) ? $ent : 'contact';
     return !isset($this->node->webform_civicrm['data'][$type][$c]);
                  */
-                self::insertComponent($field, $enabled, $this->settings, !isset($previous_data['data'][$type][$c]));
+                // Temp disabled to prevent fieldsets.
```

Comment:
If this is going to get merged like this, we should probably but a more formal `@todo` referring to an issue here?